GDJS/Runtime/gd.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for gd.js
+// Project: GDJS Platform of GDevelop
+// Definitions by: arthuro555 <https://forum.gdevelop-app.com/u/arthuro555>
+
+declare class Hashtable {
```

Comment:
While I'm not sure we should write and maintain these types for other classes because we probably can generate them, Hashtable might be an exception because ideally we want this to be a generic type: `Hashtable<gdjs.RuntimeObject>` for example.